lib/msf/base/simple/exploit.rb

```diff
@@ -147,17 +147,7 @@ def self.exploit_simple(oexploit, opts, &block)
       exploit.error = e
       exploit.print_error("Exploit failed: #{e}")
       elog("Exploit failed (#{exploit.refname}): #{e}", 'core', LEV_0)
-
-      if e.kind_of?(Msf::OptionValidateError)
-        dlog("Call stack:\n#{e.backtrace.join("\n")}", 'core', LEV_3)
-      else
-        exploit.print_error("Call stack:")
-        e.backtrace.each do |line|
-          break if line =~ /lib.msf.base.simple.exploit.rb/
-          exploit.print_error("  #{line}")
-        end
-        elog("Call stack:\n#{e.backtrace.join("\n")}", 'core', LEV_0)
-      end
+      dlog("Call stack:\n#{e.backtrace.join("\n")}", 'core', LEV_3)
```

Comment:
This line implies to me that the backtrace on this error'ed function *should* be in `framework.log`.